test-project-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+setup () {
+  # TEAM FOLDER must be in set in local environment
+  if [[ ! ${TEAM_FOLDER} ]]
+  then echo "TEAM_FOLDER missing"; exit
+  fi
+
+  export PARENT_PROJECT=$(gcloud config get-value project)
```

Comment:
So the "Parent project" is whatever project is currently active for the user? I'm not sure all the implications of a parent project, but this seems potentially wrong.

Reply:
Yes - it assumes that you already have a project with billing set up, and that it's currently active.  Not sure how to set up billing otherwise.

Reply:
Removed concept of parent project -just getting billing account.